<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BendDialog</class>
 <widget class="QDialog" name="BendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bend</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="bendTypeLabel">
       <property name="text">
        <string>Bend Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="bendTypeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="bentPitchLabel">
       <property name="text">
        <string>Bent Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="bentPitchComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="releasePitchLabel">
       <property name="text">
        <string>Release Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="releasePitchComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="bendDurationLabel">
       <property name="text">
        <string>Bend Duration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="bendDurationComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="vertStartingPointLabel">
       <property name="text">
        <string>Vertical Starting Point:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="vertStartingPointComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="vertEndingPointLabel">
       <property name="text">
        <string>Vertical Ending Point:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="vertEndingPointComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BendDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BendDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
